websites/I/Indavideó/metadata.json

```
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "195968823990550528",
+		"name": "techyyt"
+	},
+	"service": "Indavideó",
+	"description": {
+		"en": "A Hungarian video sharing site, often used to share anime."
+	},
+	"url": "indavideo.hu",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/WXPi1xl.png",
+	"thumbnail": "https://i.imgur.com/TvtJR3d.png",
+	"color": "#e74c3c",
+	"category": "videos",
+	"tags": [
+		"indavideo",
+		"hungarian",
+		"video"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*"
```

It's only one site so lets change the regex
```suggestion
	"iFrameRegExp": ".*[.]indavideo[.]hu"
```